gpii/node_modules/flowManager/src/UserLogonStateChange.js

```diff
@@ -1,8 +1,8 @@
 /*
- * GPII User Logon State Change
+ * GPII User Proximity Triggered
```

Comment:
I think the original comment was fine - there still remain endpoints related to logon state change that aren't related to proximity - as the filename suggests